web/src/components/feature.js

```diff
@@ -48,7 +48,7 @@ const StyledP = styled.p`
 export default function Feature({ bgColor, content, image }) {
     return (
         <GridWrapper bgColor={bgColor}>
-            <div className="">
+            <div className="text-wrap">
```

Comment:
Move this to a styled component with `tw` class

Reply:
@asasmith old component from OldIndex

Reply:
I'm not sure I'm following. Then why the change?